app/views/asns/show.html.erb

```diff
@@ -41,13 +41,6 @@
             <td><%= @population %></td>
           </tr>
           <tr>
-            <td><strong>Total Stake</strong></td>
```

Comment:
We do not want to remove this field, just rename it to "Stake" (see the screenshots in PT)

Reply:
Fixed.